library/blocks/block-extensions/src/main/java/org/quiltmc/qsl/block/extensions/api/client/package-info.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * <h2>Client-side Block Extensions</h2>
+ *
+ * <p>
+ * <h3>{@linkplain org.quiltmc.qsl.block.extensions.api.client.BlockRenderLayerMap Block render layer map}</h3>
+ * <ul>
+ *     <li>Provides methods for setting the render layer of
+ *     {@linkplain org.quiltmc.qsl.block.extensions.api.client.BlockRenderLayerMap#put(net.minecraft.block.Block, net.minecraft.client.render.RenderLayer) blocks}
+ *     and {@linkplain org.quiltmc.qsl.block.extensions.api.client.BlockRenderLayerMap#put(net.minecraft.fluid.Fluid, net.minecraft.client.render.RenderLayer) fluids}.</li>
```

Comment:
This should be removed since the single-block and single-fluid overloads were removed.